jerry-debugger/jerry-client-ws.py

```diff
@@ -1219,20 +1219,26 @@ def main():
             else:
                 frame_index = 0
 
+            get_total_frame = False
             while True:
 
                 buffer_pos = 3
-                while buffer_size > 0:
+                while buffer_size > 4:
                     breakpoint_data = struct.unpack(debugger.byte_order + debugger.cp_format + debugger.idx_format,
                                                     data[buffer_pos: buffer_pos + debugger.cp_size + 4])
 
-                    breakpoint = get_breakpoint(debugger, breakpoint_data)
-
-                    print("Frame %d: %s" % (frame_index, breakpoint[0]))
-
-                    frame_index += 1
-                    buffer_pos += 6
-                    buffer_size -= 6
+                    if get_total_frame:
+                        breakpoint = get_breakpoint(debugger, breakpoint_data)
+                        print("Frame %d: %s" % (frame_index, breakpoint[0]))
+                        buffer_pos += 6
+                        buffer_size -= 6
+                        frame_index += 1
+                    else:
+                        total_frames = breakpoint_data[0]
+                        get_total_frame = True
+                        buffer_pos += 4
+                        buffer_size -= 4
+                        print("Total Frames: %d" % total_frames)
```

Comment:
`print("Total number of frames: %d" % total_frames)`

BTW, do we really need to print this information in the python client? I think it would be better to ignore it.

Reply:
I thought it might make it easier since you'll know how many frames can you choose from.

Reply:
I don't mind to keep it. It is only my personal preference.

What do you think about this?
1) Omit the total number of frames when we get the full backtrace. 
```
> backtrace
frame 0: ...
frame 1: ...
....
frame N: ...
```
2) Print the remaining frames when we get the first `X` frames.
```
> backtrace X
Getting the first X frames. There are (N - X) frames left.
frame 0: ...
frame 1: ...
....
frame X: ...
```
3) Printing the number of total and got frames.
```
> backtrace X Y
Getting (Y - X) frame out of N.
frame X: ...
frame X + 1: ...
....
frame Y: ...
```

Reply:
I would add it at the end:
```
frame 0:
frame 1:
There are 7 frames left.
```

Reply:
@LaszloLango That does seem nice. Added it. :)